src/views/studio/MyVideosView/MyVideos.styles.ts

```diff
@@ -39,3 +42,55 @@ export const SortContainer = styled.div`
     display: grid;
   }
 `
+
+export const NewVideoTile = styled(Link)`
+  position: relative;
+  background: none;
+  border: 1px dashed ${colors.gray[500]};
+  color: ${colors.gray[500]};
+  transition: border ${transitions.timings.player} ease-out;
+  width: 100%;
+  height: 0;
+
+  /* 2px account for border */
+  padding-bottom: calc(56.25% - 2px);
+  cursor: pointer;
+
+  :hover {
+    border: 1px dashed ${colors.gray[200]};
+    color: ${colors.gray[200]};
+    ${() => StyledIcon} {
+      > path {
+        fill: ${colors.gray[200]};
+      }
+    }
+    ${() => StyledText} {
```

Comment:
If we can, let's avoid this type of selectors. In this case I think you can just move `StyledIcon` and `StyledText` above this declaration, right?